Documentation/building/windows/dependencies.md

```diff
@@ -0,0 +1,44 @@
+# Build Dependencies for Windows
+
+Building Xamarin.Android requires:
+
+  * An existing installation of the Xamarin.Android SDK and the Android SDK
```

Comment:
Why is this required?

Reply:
I think nuget restore would likely fail.

This change might help this: https://github.com/xamarin/xamarin-android/commit/46f02511f8c91b133fb09fdcad2fa8fe9f2cc730

But it is only on master.

Reply:
If this is still the case, please file a bug.  We should *not* require a pre-existing system install of Xamarin.Android in order to build xamarin-android.

Additionally, if it *does* work on master, that's *fine*, as these docs are going into master. :-)